tools/lsmcli/cmdline.py

```diff
@@ -1822,7 +1822,9 @@ def local_disk_list(self, args):
                     info_dict[key] = func_dict[key](disk_path)
                 except LsmError as lsm_err:
                     if lsm_err.code != ErrorNumber.NO_SUPPORT:
-                        raise
+                        print("BUG: %s('%s'): %d %s" %
```

Comment:
Calling errors here a bug is a little overdoing it from my perspective. It'll make the customer think something is wrong with the software, even if the errors is with unsupported hardware (like an SD card, which we ran into). I would simply cut off the "BUG: " portion of your current message.

Reply:
I changed the word "BUG" to "WARN" in new patch.